include/ezc3d.h

```diff
@@ -204,10 +198,10 @@ class EZC3D_API ezc3d::c3d {
 
 protected:
     // Internal reading and writting function
-    char * c_float; ///< Char to be used by the read function with the specific size of a float preventing to allocate it at each calls
-    char * c_float_tp; ///< Char to be used by the read function with the specific size of a float preventing to allocate it at each calls (allow for copy of c_float)
-    char * c_int; ///< Char to be used by the read function with the specific size of a int preventing to allocate it at each calls
-    char * c_int_tp; ///< Char to be used by the read function with the specific size of a int preventing to allocate it at each calls  (allow for copy of c_int)
+    std::vector<char>(c_float); ///< Char to be used by the read function with the specific size of a float preventing to allocate it at each calls
+    std::vector<char>(c_float_tp); ///< Char to be used by the read function with the specific size of a float preventing to allocate it at each calls (allow for copy of c_float)
+    std::vector<char>(c_int); ///< Char to be used by the read function with the specific size of a int preventing to allocate it at each calls
+    std::vector<char>(c_int_tp); ///< Char to be used by the read function with the specific size of a int preventing to allocate it at each calls  (allow for copy of c_int)
```

Comment:
The parantheses are not needed. Actually, they throw warnings on my compiler with my flags:

```
include/ezc3d.h:201:22: warning: unnecessary parentheses in declaration of 'c_float' [-Wparentheses]
     std::vector<char>(c_float); ///< Char to be used by the read function with the specific size of a float preventing to allocate it at each calls
```